users/permissions.py

```diff
@@ -6,11 +7,14 @@
 from users.common.strings import PermissionsMessage
 from users.models import CustomUser
 
+logger = logging.getLogger(__name__)
+
 
 class AuthenticatedAdmin(permissions.BasePermission):
     message = PermissionsMessage.NONE_ADMIN_USER
 
     def has_permission(self, request: HttpRequest, _view: Any) -> bool:  # pylint: disable=no-self-use
+        logger.info(f"Checking permission for User: {request.user} - AuthenticatedAdmin")
```

Comment:
Instead of that I would add log only when user doesn't have permission, as it is not ordinary situation.

Reply:
Done